src/envoy/http/authn/http_filter.cc

```diff
@@ -118,17 +106,6 @@ void AuthenticationFilter::setDecoderFilterCallbacks(
   decoder_callbacks_ = &callbacks;
 }
 
-void AuthenticationFilter::continueDecoding() {
-  if (state_ != State::PROCESSING) {
-    ENVOY_LOG(error, "State {} is not PROCESSING.", state_);
-    return;
-  }
-  state_ = State::COMPLETE;
-  if (stopped_) {
-    decoder_callbacks_->continueDecoding();
-  }
-}
-
 void AuthenticationFilter::rejectRequest(const std::string& message) {
```

Comment:
I think we just need sendLocalReply part. There is no point of keeping state_ any more. All other function like decodeTrailers can just return continue. Anway, @qiwzhang may know better; can you confirm?